chroma-manager/tests/framework/integration/existing_filesystem_configuration/cluster_setup

```diff
@@ -172,6 +172,7 @@ source $CHROMA_DIR/chroma-manager/tests/framework/integration/utils/install_clie
 # Install and setup integration tests
 if ! $JENKINS; then
     CMIT=$(ls chroma-manager/dist/chroma-manager-integration-tests-*.x86_64.rpm)
+    [[ -z "$CMIT" ]] && (echo "Could not find chroma-manager/dist/chroma-manager-integration-tests package"; exit 1)
```

Comment:
Not only is it more idiomatic to test the return value of `ls` rather than the "emptyness" of the LHS of the assignment, our use of `set -e` will actually make this script exit if the `ls` fails and the error message will never be printed.

The above two lines should be changed to:

```
if ! CMIT=$(ls chroma-manager/dist/chroma-manager-integration-tests-*.x86_64.rpm 2>/dev/null); then
    echo "Could not find chroma-manager/dist/chroma-manager-integration-tests package"
    exit 1
```

Reply:
where is the ```set -e``` that is active during this line? I can't see it

Reply:
changed it anyway

Reply:
To answer the question, it's right on [the first line](https://github.com/intel-hpdd/intel-manager-for-lustre/blob/4146442dae61938ee046fce56662cc50f607c8aa/chroma-manager/tests/framework/integration/existing_filesystem_configuration/cluster_setup#L1) of the file:

```
#!/bin/bash -ex
```